benchmarks/on-policy/README.md

```diff
@@ -48,14 +67,45 @@ if __name__ == '__main__':
     naive_lagrange_policy = ['PPOLag', 'TRPOLag', 'RCPO', 'OnCRPO', 'PDO']
     first_order_policy = ['CUP', 'FOCOPS', 'P3O']
     second_order_policy = ['CPO', 'PCPO']
-
-    eg.add('algo', base_policy + naive_lagrange_policy + first_order_policy + second_order_policy)
+    saute_policy = ['PPOSaute', 'TRPOSaute']
+    simmer_policy = ['PPOSimmerPID', 'TRPOSimmerPID']
+    early_mdp_policy = ['PPOEarlyTerminated', 'TRPOEarlyTerminated']
+
+    eg.add(
+        'algo',
+        base_policy +
+        naive_lagrange_policy +
+        first_order_policy +
+        second_order_policy +
+        saute_policy +
+        simmer_policy +
+        early_mdp_policy
+    )
 
     # you can use wandb to monitor the experiment.
     eg.add('logger_cfgs:use_wandb', [False])
     # you can use tensorboard to monitor the experiment.
     eg.add('logger_cfgs:use_tensorboard', [True])
 
+	# if you want to reproduce results of 1e6, using
+	# eg.add('algo_cfgs:steps_per_epoch', [2048])
+	# eg.add('train_cfgs:total_steps', [2048 * 500])
```

Comment:
it seems some mismatched with indent, because I type it just in the comment box.

Reply:
This may be a rendering problem of github website, it seems no indent in readme file.